spec/unit/navigation/navigation.spec.js

```diff
@@ -135,8 +135,10 @@ describe("navigation toggle", () => {
 
   it("collapses dropdowns when the Escape key is hit", () => {
     accordionButton.click();
-    EVENTS.escape(accordionButton);
-    assert.strictEqual(accordionButton.getAttribute("aria-expanded"), "false");
+    setTimeout(() => {
```

Comment:
Mocha won't wait for this timeout unless you add `done` as an argument and call it in the callback, which can cause some strange side effects in subsequent tests.

https://mochajs.org/#asynchronous-code

Reply:
you're right  @aduth, will take another look